sandboxes/andromeda/consumer.cannonfile.toml

```diff
@@ -0,0 +1,288 @@
+target = ["andromeda_sandbox.synthetix.USDProxy"]
+from = "<%= settings.test_user %>"
+func = "approve"
+args = [
+    "<%= imports.andromeda_sandbox.imports.perps_factory.contracts.PerpsMarketProxy.address %>",
+    "<%= parseEther(String(1_000)) %>",
+]
+depends = ["invoke.create_perps_account"]
+
+[invoke.deposit_1k_susd_to_perps]
+target = ["andromeda_sandbox.perps_factory.PerpsMarketProxy"]
+from = "<%= settings.test_user %>"
+func = "modifyCollateral"
+args = [
+    "<%= extras.test_user_perps_account_id %>",
+    0, # 0 is a special market id for susd
+    "<%= parseEther(String(1_000)) %>",
+]
+depends = ["invoke.approve_1k_susd_for_PerpsMarketProxy_to_trade"]
+
+#
+#
+#
+# Withdrawing 300 sUSD from perps to get 300 USDC back into the wallet
+#
+#
+#
+[invoke.approve_700_susd_for_PerpsMarketProxy_to_withdraw]
+target = ["andromeda_sandbox.synthetix.USDProxy"]
+from = "<%= settings.test_user %>"
+func = "approve"
+args = [
+    "<%= imports.andromeda_sandbox.imports.perps_factory.contracts.PerpsMarketProxy.address %>",
+    "<%= parseEther(String(700)) %>",
+]
+depends = ["invoke.deposit_1k_susd_to_perps"]
+
+[invoke.withdraw_300_susd_from_perps]
+target = ["andromeda_sandbox.perps_factory.PerpsMarketProxy"]
+from = "<%= settings.test_user %>"
+func = "modifyCollateral"
+args = [
+    "<%= extras.test_user_perps_account_id %>",
+    0, # 0 is a special market id for susd
+    "<%= parseEther(String(700)) %>",
+]
+depends = ["invoke.approve_700_susd_for_PerpsMarketProxy_to_withdraw"]
+
+[invoke.approve_300_susd_for_SpotMarketProxy_to_buy_synth_usdc]
+target = ["andromeda_sandbox.synthetix.USDProxy"]
+from = "<%= settings.test_user %>"
+func = "approve"
+args = [
+    "<%= imports.andromeda_sandbox.imports.spot_factory.contracts.SpotMarketProxy.address %>",
+    "<%= parseEther('300') %>",
+]
+depends = ["invoke.withdraw_300_susd_from_perps"]
+
+[invoke.atomic_buy_300_synth_usdc_with_300_susd]
+target = ["andromeda_sandbox.spot_factory.SpotMarketProxy"]
+from = "<%= settings.test_user %>"
+func = "buy"
+args = [
+    # uint128 marketId
+    "<%= imports.andromeda_sandbox.extras.synth_usdc_market_id %>",
+    # uint256 usdAmount
+    "<%= parseEther('300') %>",
+    # uint256 minAmountReceived (0% slippage!)
+    "<%= parseEther('300') %>",
+    # address referrer
+    "<%= AddressZero %>"
+]
+depends = ["invoke.approve_300_susd_for_SpotMarketProxy_to_buy_synth_usdc"]
+
+[invoke.unwrap_300_usdc_from_synth_usdc]
+target = ["andromeda_sandbox.spot_factory.SpotMarketProxy"]
+from = "<%= settings.test_user %>"
+func = "unwrap"
+args = [
+    # uint128 marketId
+    "<%= imports.andromeda_sandbox.extras.synth_usdc_market_id %>",
+    # uint256 unwrapAmount
+    "<%= parseEther('300') %>",
+    # uint256 minAmountReceived (0% slippage!)
+    "<%= parseEther('300') %>"
+]
+depends = ["invoke.atomic_buy_300_synth_usdc_with_300_susd"]
+
+#
+#
+#
+# Trading perps
+# TODO: figure out order settlements and closing orders
```

Comment:
I think until we have order settlement figured out we probably should avoid commitOrder. This order will go stale and require users to manually cancel before they can commit a new order

Reply:
We have it figured out but it cannot be done in toml file unfortunately. I will comment out this code yes.